### DISCUSSION TITLE
An error about failed to resolve entry for package

### DISCUSSION THREAD

**Discussion by rrriverrain** [vitest-dev/vitest#5553] [Q&A]
when I tried to run my tests, there was always an error :"Failed to resolve entry for package "monaco-editor". The package may have incorrect main/module/exports specified in its package.json."I have on idea about this , so I need some help , would somebody hepl me . (my node version 18.14.0 ,vitest version 0.33.0 , vite version 4.5.0)

**Comment by Habibulloh08** [comment]
/// <reference types="vitest" />
/// <reference types="vite/client" />
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
  test: {
    globals: true,
    environment: "jsdom",
    setupFiles: "./src/tests/setup.ts",
  },
});
try writing these codes in vitest config file if the problem is not solved https://vitest.dev/config/
you can get the answer from vite page
